cmd/convox/templates/init/rails/bin/web

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
When you volume mount the app directory the file from the image becomes unavailable because it isn't present on the host.

I will also send a PR to the Docker image repo for this.